src/langchain_google_alloydb_pg/vectorstore.py

```diff
@@ -341,6 +343,39 @@ async def aadd_documents(
         ids = await self.aadd_texts(texts, metadatas=metadatas, ids=ids, **kwargs)
         return ids
 
+    def encode_image(self, uri: str) -> str:
+        """Get base64 string from image URI."""
+        with open(uri, "rb") as image_file:
+            return base64.b64encode(image_file.read()).decode("utf-8")
+
+    async def aadd_images(
+        self,
+        images: List[str],
+        metadatas: Optional[List[dict]] = None,
+        ids: Optional[List[str]] = None,
+        **kwargs: Any,
+    ) -> List[str]:
+        """Embed images and add to the table."""
+        encoded_images = []
+        if metadatas is None:
+            metadatas = [{"image_uri": uri} for uri in images]
+
+        for uri in images:
+            encoded_image = self.encode_image(uri)
+            encoded_images.append(encoded_image)
+
+        if hasattr(self.embedding_service, "embed_image"):
+            embeddings = self.embedding_service.embed_image(images)
```

Comment:
We should prepare some documentation for how to use this functionality. Current vertexaiembedding doesn't work since we don't have support for a list

Reply:
For now we might want to do a try/except to ensure all Embedding service work here until we sort out the VertexAIEmbedding changes

Reply:
added try/except statement